relay-general/src/store/normalize/span/description.rs

```diff
@@ -534,28 +537,28 @@ mod tests {
 
     span_description_test!(
         span_description_dont_scrub_double_quoted_strings_format_postgres,
-        r#"SELECT * from \"table\" WHERE sku = %s"#,
+        r#"SELECT * from "table" WHERE sku = %s"#,
```

Comment:
`span_description_test!` previously expected JSON encoded strings, hence the escape characters. Now it takes pure strings.